test/scripts/deposit.js

```diff
@@ -0,0 +1,74 @@
+const Contract = require("@truffle/contract")
+
+const BatchExchange = Contract(require("@gnosis.pm/dex-contracts/build/contracts/BatchExchange"))
+const GnosisSafe = artifacts.require("GnosisSafe")
+const ProxyFactory = artifacts.require("GnosisSafeProxyFactory")
+
+const { addCustomMintableTokenToExchange, deploySafe } = require("../test_utils")
+const { deployFleetOfSafes, buildDepositFromList } = require("../../scripts/utils/trading_strategy_helpers")(web3, artifacts)
+const { waitForNSeconds, execTransaction } = require("../../scripts/utils/internals")(web3, artifacts)
+
+contract("Deposit scripts", function (accounts) {
+  let gnosisSafeMasterCopy
+  let proxyFactory
+  let exchange
+  const safeOwner = accounts[0]
+  beforeEach(async function () {
+    gnosisSafeMasterCopy = await GnosisSafe.new()
+    proxyFactory = await ProxyFactory.new()
+
+    BatchExchange.setProvider(web3.currentProvider)
+    exchange = await BatchExchange.deployed()
+  })
+
+  const setup = async function (numberOfBrackets, amounts) {
+    const masterSafe = await GnosisSafe.at(await deploySafe(gnosisSafeMasterCopy, proxyFactory, [safeOwner], 1))
+    const bracketAddresses = await deployFleetOfSafes(masterSafe.address, numberOfBrackets)
+    const tokenInfo = []
+    const deposits = []
+    for (const { tokenData = {}, amount } of amounts) {
+      const symbol = tokenData.symbol || "TEST"
+      const decimals = tokenData.decimals || 18
+      // Adding token to exchange is not actually necessary since one can deposit and withdraw any token.
+      const { id, token } = await addCustomMintableTokenToExchange(exchange, symbol, decimals, accounts[0])
+      tokenInfo.push({ symbol: symbol, decimals: decimals, id: id, token: token, address: token.address })
```

Comment:
This should also work:
```suggestion
      tokenInfo.push({ symbol, decimals, id, token, address: token.address })
```
But feel free to leave it as it was.

Reply:
I like it.